eng/pipelines/templates/jobs/perf.yml

```diff
@@ -23,6 +23,9 @@ parameters:
   - name: additionalArguments
     type: string
     default: ''
+  - name: disableAzureResourceCreation
```

Comment:
we can keep it if it's useful, but my intention is to eventually deploy resources, so empty template would be as good as this.
For starter I'm planning to add storage account with container accessible via sas url.

Reply:
I will remove this and add the empty bicep file